packages/frontend-api/src/Resources/config/graphql-types/ProductFilter/ProductFilterOptionsDecorator.types.yaml

```diff
@@ -0,0 +1,18 @@
+ProductFilterOptionsDecorator:
+    type: interface
+    decorator: true
+    config:
+        description: "Represents a product filter options"
+        fields:
+            minimalPrice:
+                type: "Money!"
+            maximalPrice:
+                type: "Money!"
+            brands:
+                type: "[BrandFilterOption]"
+            flags:
+                type: "[FlagFilterOption]!"
```

Comment:
I'm confused about mandatory flags, is it intentional?

lists in this definition file should not allow nulls by adding ! before [ 
like `[BrandFilterOption!]`

Reply:
190f141abae92afe3f2fc09f1f542853f4c79247